pallets/communities/Cargo.toml

```diff
@@ -9,6 +9,8 @@ license = "MIT-0"
 publish = false
 repository = "https://github.com/substrate-developer-hub/substrate-node-template/"
 
+[workspace]
```

Comment:
is this workspace definition necessary?

Reply:
It's a workaround to have the rust-analyzer working on VSCode. I was actually going to ask you how would you do that on your env? 😅